query.go

```diff
@@ -369,6 +369,56 @@ func labelFilter(target string, useSep bool) driver.FilterFn {
 	})
 }
 
+func GetName(info driver.Info) (name string, id string) {
```

Comment:
Exposed this because the RDK does this exact thing. I'll update that code to use this function instead so we don't duplicate logic.